python/tvm/driver/tvmc/compiler.py

```diff
@@ -185,13 +185,21 @@ def compile_model(
     """
     dump_code = [x.strip() for x in dump_code.split(",")] if dump_code else None
     mod, params = frontends.load_model(path, model_format, shape_dict)
+    config = {}
 
     if alter_layout:
         mod = common.convert_graph_layout(mod, alter_layout)
 
-    tvm_target = common.target_from_cli(target)
+    tvm_target, extra_targets = common.target_from_cli(target)
     target_host = tvm_target if not target_host else target_host
 
+    for codegen_from_cli in extra_targets:
+        codegen = composite_target.get_target_by_name(codegen_from_cli["name"])
```

Comment:
It's weird to see "get_target_by_name" returns a "codegen". Maybe `get_codegen_by_target`?

Reply:
updated in the last version of the PR